File: app/src/main/java/workshop1024/com/xproject/XApplication.java
package workshop1024.com.xproject;

import android.app.Application;

import workshop1024.com.xproject.utils.ThemeUtils;

public class XApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        ThemeUtils.refreshThemesShow(this);
    }
}

File: android/app/src/main/java/workshop1024/com/xproject/XApplication.kt
package workshop1024.com.xproject

import android.app.Application
import com.facebook.soloader.SoLoader
import workshop1024.com.xproject.utils.ThemeUtils

class XApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        ThemeUtils.refreshThemesShow(this)

        SoLoader.init(this,false)
    }
}